mono/tests/test-runner.cs

```diff
@@ -205,8 +205,12 @@ class TestInfo {
 
 		if (!String.IsNullOrEmpty (inputFile)) {
 			foreach (string l in File.ReadAllLines (inputFile)) {
-				for (int r = 0; r < repeat; ++r)
-					tests.Add (l);
+				foreach (string m in l.Split (' ')) {
+					if (!String.IsNullOrEmpty (m)) {
```

Comment:
StringSplitOptions.RemoveEmptyEntries?

Reply:
I agree, but it's also not worth spinning another build as this is test code so we can be a little more lenient :)